### PR TITLE
fix(input): update helper text color for ionic theme

### PR DESCRIPTION
**Pull request by thetaPC**
Issue number: internal

---------

## What is the current behavior?

The helper text has already been implemented. However, the color does not match the [design](https://www.figma.com/file/Fun8j9vlydE65HixgBpA7N/%F0%9F%A5%B7-App-X---UI-Components---WIP?type=design&node-id=18966-451747&mode=design&t=Vrj1446NKDnLkFJD-0) for the `ionic` theme.

## What is the new behavior?

- The helper text color has been updated to match the design for the `ionic` theme.
- The bottom content input test has been adjusted to include `ionic-md`.
- The color was hard coded until the `ionic` design tokens are set.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

The input snapshots look plain because this PR only focuses on the helper text.

### CODE REVIEW

File: core/src/components/input/test/bottom-content/input.e2e.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, config }) =>
 /**
  * Rendering is the same across modes
  */
-configs({ modes: ['md'], directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
+configs({ modes: ['md', 'ionic-md'], directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
```
Comment (thetaPC) [inline core/src/components/input/test/bottom-content/input.e2e.ts:56]: I decided to reuse these tests since at some point, the `ionic` theme will implement the helper text and error text and these tests are testing exactly what needs to be checked.

File: core/src/components/input/test/bottom-content/input.e2e.ts-snapshots/input-bottom-content-error-ionic-md-ltr-light-Mobile-Chrome-linux.png
Comment (thetaPC) [inline core/src/components/input/test/bottom-content/input.e2e.ts-snapshots/input-bottom-content-error-ionic-md-ltr-light-Mobile-Chrome-linux.png:1]: This screenshots were generated because `ionic-md` was added to an existing test. These screenshots will be changed through a different PR that works on the error text.

File: core/src/components/input/test/bottom-content/input.e2e.ts [hunk 2]
```diff
@@ -36,6 +36,31 @@ configs({ directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
   });
 });
 
+configs({ directions: ['ltr'], modes: ['ionic-md'] }).forEach(({ title, screenshot, config }) => {
+  test.describe(title('input: bottom content'), () => {
```
Comment (thetaPC) [inline core/src/components/input/test/bottom-content/input.e2e.ts:40]: `ionic` theme input does not have a solid fill so we don't test it.

File: core/src/components/input/test/bottom-content/input.e2e.ts-snapshots/input-bottom-content-error-ionic-md-ltr-light-Mobile-Chrome-linux.png
Comment (thetaPC) [inline core/src/components/input/test/bottom-content/input.e2e.ts-snapshots/input-bottom-content-error-ionic-md-ltr-light-Mobile-Chrome-linux.png:1]: The error text screenshots are being generated because `ionic-md` was added to an existing test. These screenshots will be updated through another PR that implements the `ionic` theme for the error text.

File: core/src/components/input/test/bottom-content/input.e2e.ts-snapshots/input-error-custom-color-ionic-md-ltr-light-Mobile-Chrome-linux.png
Comment (thetaPC) [inline core/src/components/input/test/bottom-content/input.e2e.ts-snapshots/input-error-custom-color-ionic-md-ltr-light-Mobile-Chrome-linux.png:1]: The error text screenshots are being generated because `ionic-md` was added to an existing test. These screenshots will be updated through another PR that implements the `ionic` theme for the error text.

**Review by sean-perkins — APPROVED** [review]
